### DISCUSSION TITLE
Make labels clearer on Delete Predictions beyond Frame Limit feature

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#2077] [Ideas]
In https://github.com/talmolab/sleap/pull/1797 we added the option to delete instances beyond a frame range. We should add more clarity in the dialog to make it clear that this deletes instances BEFORE and AFTER the range, not INSIDE of it.

**Comment by gitttt-1234** [comment]
closing this discussion, the labels are fixed in #2711
